src/main/java/frc/robot/commands/ResetYawCommand.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.commands;
```

Comment:
I think this is good practice to learn how to create commands. But generally, instead of creating a two line command. We can just create a method in the drive subsystem in this case.